packages/block-editor/src/components/story-update-progress/README.md

```diff
@@ -0,0 +1,105 @@
+StoryUpdateProgress
+===================
+
+`StoryUpdateProgress` shows a progress bar while the media files associated with a Story block are being saved and uploaded
```

Comment:
Same than what I mentioned [here.](https://github.com/WordPress/gutenberg/pull/25242/files#discussion_r501515233)

It's possible that this component could give us troubles, since it's defined on the `gutenberg` repo but used only on a `jetpack` block and referring to a jetpack/wpcom feature (Stories).

What I would recommend in this case is to make all required bridge methods generic, and move this component to Jetpack.

Pinging @hypest and @maxme to get their opinion, since this would be a bigger refactor.

Reply:
As discussed elsewhere with @etoledom - I'll make this component generic given it may be used on other blocks containing a collection of media in the future, and that will solve this situation 👍

Reply:
Addressed in the set of PRs mentioned / starting with this https://github.com/WordPress/gutenberg/pull/26005